scss/gallery/_media-attachment.scss

```diff
@@ -146,3 +146,52 @@
     }
```

Comment:
Non-blocking: for documents: would be nice if the cursor became a pointer when hovering over the title to make clear that it is clickable as well.
Analogically for webxdc content, pointer would be nice.